benches/profile/test_worker_proxy.py

```
@@ -0,0 +1,221 @@
+            def poll(self):
+                return None
+
+            def terminate(self):
+                raise ProcessLookupError("already exited")
+
+        terminate_child(ExitedChild())
+
+    def test_child_may_exit_while_proxy_stdin_remains_open(self):
+        proxy = pathlib.Path(__file__).with_name("worker_proxy.py")
+        env = dict(os.environ, KAKEHASHI_WORKER_PROXY_BIN=sys.executable)
+        process = subprocess.Popen(
```

Ruff (0.15.21)

[error] `subprocess` call: check for execution of untrusted input

(S603)